### PR TITLE
docs: clean github templates

### PR DESCRIPTION
**Pull request by markov00**
## Summary

A small cleanup on the github templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -40,5 +40,8 @@ Add any [Kibana](https://github.com/elastic/kibana) related issues here.
 
 
 **Checklist**
+
+Delete any items that are not applicable to this issue.
```
Comment (nickofthyme) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Thank god!

**Comment by markov00** [comment]
cla/check

**Pull request merged**